dts/arm/nordic/nrf54l_common.dtsi

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include "nrf_common.dtsi"
+
+/ {
+	rram_controller: rram-controller@5004b000 {
+		compatible = "nordic,rram-controller";
+		reg = <0x5004b000 0x1000>;
+		#address-cells = <1>;
+		#size-cells = <1>;
+		interrupt-parent = <&nvic>;
+		interrupts = <75 NRF_DEFAULT_IRQ_PRIORITY>;
+		rram0: rram@0 {
+			compatible = "soc-nv-flash";
+			erase-block-size = <4096>;
+			write-block-size = <1>;
+		};
+	};
+
+	chosen {
+		zephyr,flash-controller = &rram_controller;
+	};
```

Comment:
avoid choices in soc dts files

Reply:
It is also in dts files for other platforms, for example in `dts/arm/nordic/nrf5340_cpuappns.dtsi`. Should I move it anyway to the boards folder?